AudioAnalytics/AudioAnalyzer/src/main/java/sample/AudioAnalyzer.java

```diff
@@ -114,50 +103,39 @@ private static void runAnalyzer(ConfigWrapper config) {
                     outputAssetName);
 
             Job job = submitJob(manager, config.getResourceGroup(), config.getAccountName(),
-                audioAnalyzerTransform.name(), jobName, inputAssetName, outputAsset.name());
+                    audioAnalyzerTransform.name(), jobName, inputAssetName, outputAsset.name());
 
             long startedTime = System.currentTimeMillis();
-            
-            EventProcessorHost eventProcessorHost = null;
+
             try {
                 // First we will try to process Job events through Event Hub in real-time. If this fails for any reason,
                 // we will fall-back on polling Job status instead.
                 System.out.println("Creating an event processor host to process events from Event Hub...");
-
                 String storageConnectionString = "DefaultEndpointsProtocol=https;AccountName=" +
-                    config.getStorageAccountName() +
-                    ";AccountKey=" + config.getStorageAccountKey();
-                
+                        config.getStorageAccountName() +
```

Comment:
You can get your connection string from the portal instead of constructing it like this. 
![image](https://user-images.githubusercontent.com/10136526/133184367-31446613-f216-4efb-a562-8d243e94054b.png)

Reply:
According to your comments, it has been fixed in the new version.